sources/packages/backend/apps/api/src/route-controllers/student-appeal/models/student-appeal.model.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Student appeal form data which is not persisted
+ * but used for logic or validation purpose.
+ */
+export class StudentAppealNonSubmissionData {
```

Comment:
`NonSubmission` can be misleading because `programYear` is part of the submitted payload even if not part of the persisted data.
I would suggest `StudentAppealServerSideSubmissionData` or just `StudentAppealServerSideData`.